google-apis.md

```diff
@@ -0,0 +1,22 @@
+# ☁️ Google APIs (Maps, Geocoding)
+
+At DataMade, we use several Google APIs to support our maps. Google continues to have the most robust [Geocoding service](https://developers.google.com/maps/documentation/geocoding/start) and we often times use the [Google Maps Javascript API](https://developers.google.com/maps/documentation/javascript/tutorial) for their map tiles as well.
+
+To keep accurate accounting of API usage per website, we create a separate Google API key for each project.
+
+DataMade staff have access to these keys here: https://console.developers.google.com/apis/credentials?folder=&organizationId=&project=adroit-hall-205721
```

Comment:
Is there any downside to making this link public?

Reply:
I think it would be fine and the URL is not related directly to any of our API keys, plus you need a login to access.